requirements.txt

```diff
@@ -1,14 +1,9 @@
-backports.functools_lru_cache==1.6.1; python_version == "2.7"
 dateparser==0.7.*
-pre-commit==1.20.*; python_version <= "3.5"
-pre-commit==2.3.*; python_version >= "3.6"
-pytest==4.6.*; python_version == "2.7"
-pytest==5.4.*; python_version >= "3.5"
-pytest-cov==2.8.*
-pytest-mock==2.0.*; python_version == "2.7"
-pytest-mock==3.1.*; python_version >= "3.5"
+pre-commit==2.4.*
```

Comment:
Do you think we should add `mypy` in here as well?

Reply:
yes indeed, I forgot to somehow!

Reply:
Added mypy via pre-commit, it installs it in a separate virtualenv, so it doesn't affect main requirements, for dev purposes, maybe we can add requirements.dev.txt?